CONTRIBUTING.md

```diff
@@ -76,15 +76,45 @@ To address feedback from code review, implement changes and push to your feature
 ### Code style and formatting
 The Lingua Franca compiler is implemented in Java and Kotlin. The overarching advice is to use each language's most widely used idioms and conventions, which are fortunately very similar. The code base is shipped with a [Spotless](https://github.com/diffplug/spotless) configuration to check and enforce style compliance. Lingua Franca code (e.g., tests) in this repository is also automatically formatted via Spotless.
 
+
+#### Formatting
+
 - To check that modified files are formatted correctly, run:
+
 ```
 ./gradlew spotlessCheck
 ```
+
 - To apply the changes recommended by the formatter, run:
+
 ```
 ./gradlew spotlessApply
 ```
 
+_Java code formatting in IntelliJ_
+
+Follow the directions here to install the Spotless Gradle plugin. We are not currently aware of a way to run Spotless
```

Comment:
Looks like this is incomplete. I didn't see a description of the plugin in the diff.

Reply:
Thanks for the catch! I just added a link.